python_modules/dagster/dagster/_core/definitions/events.py

```diff
@@ -189,6 +190,24 @@ def from_coercible_or_definition(
         else:
             return AssetKey.from_coercible(arg)
 
+    @staticmethod
+    def from_coercible_to_asset_dep(arg: "CoercibleToAssetDep") -> "AssetKey":
```

Comment:
Name should be `from_coercible_to_asset_key`?

Reply:
ah ok i was wondering if this name would be confusing, and it is. my intention was "from CoercibleToAssetDep" not "from Coercible to AssetDep".

The existing "coercible" methods are:
* `from_coercible` which does `CoercibleToAssetKey` -> `AssetKey`
* `from_coercible_or_definition` which does `CoercibleToAssetKey | AssetsDefinition | SourceAsset` -> `AssetKey`

I'm unsure how to keep consistency with this pattern, while not introducing confusion with the method name. i'll ponder and respond with ideas